src/toil/provisioners/gceProvisioner.py

```diff
@@ -34,13 +34,24 @@
 logging.getLogger("urllib3.connectionpool").setLevel(logging.WARNING)
 
 
+# pulled from stack overflow
+# fix for objects getting passed to a json serializer which dies on bytes
+def convert_bytes(data):
+    if isinstance(data, bytes):  return data.decode('ascii')
```

Comment:
This should be `data.decode('utf-8')` instead of `ascii`, unless there's a compelling reason not to.